src/GovUk.Education.ExploreEducationStatistics.Admin/Validators/ReleaseRequestValidators.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.AspNetCore.Mvc;
+using GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationErrorMessages;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Validators;
+
+public static class ReleaseCreateRequestValidator
```

Comment:
It's nice to have the validation very clearly available like this :)

Reply:
Just wondering, and it doesn't have to be part of this PR, but if it's possible to use validation attributes on the Request objects themselves to achieve this same behaviour?  Might be one to put on a back burner until we get round to getting FluentValidation into the mix properly though.

Reply:
I can't find one, but maybe there is? Would be interested to know if anyone knows about it

I discussed putting these validators in their own file/class with Ben. I suggested that it might be a stepping stone to moving towards using FluentValidation. We have these dedicated classes for validating which we can then hook into when we start to use FluentValidation - we just make the classes extend `AbstractValidator<T>` and make any necessary changes. 

Would be good to get some wider opinions on this too.

This is what made it a necessity to add the new `Either` extension method which validates the request object first, up-front, before making any DB calls to check for Entity existence etc...